pandas/core/indexers.py

```diff
@@ -66,28 +70,25 @@ def check_setitem_lengths(indexer, value, values) -> None:
     """
     Validate that value and indexer are the same length.
 
-    An special-case is allowed for when the indexer is a boolean array
-    and the number of true values equals the length of ``value``. In
-    this case, no exception is raised.
+    A special-case is allowed for when the indexer is a
```

Comment:
I think this section in particular read better with the previous text, so OK to revert

Reply:
I think should revert altogether. not explicitly checked here but our docstring validator requires a blank line between short summary and extended description, which this does away with

The old formatting was fine so can just leave as is

Reply:
This is still showing a diff.

Reply:
> This is still showing a diff.

Just because I changed the ```An``` to ```A``` at the very first line